### PR TITLE
Openstack: describes how to enable qemu agent for UPI.

### PR DESCRIPTION
**Pull request by adduarte**
implements [OSASINFRA-2191](https://issues.redhat.com/browse/OSASINFRA-2191)

### CODE REVIEW

File: docs/user/openstack/install_upi.md [hunk 1]
```diff
@@ -207,6 +207,16 @@ $ openstack image create --container-format=bare --disk-format=qcow2 --file rhco
 
 **NOTE:** Depending on your OpenStack environment you can upload the RHCOS image as `raw` or `qcow2`. See [Disk and container formats for images](https://docs.openstack.org/image-guide/introduction.html#disk-and-container-formats-for-images) for more information.
 
+[qemu_guest_agent]: https://docs.openstack.org/nova/latest/admin/configuration/hypervisor-kvm.html 
+If the rhcos image being used supports it,  the [KVM Qemu Guest Agent][qemu_guest_agent] may be used to enable optional
```
Comment (mandre) [inline docs/user/openstack/install_upi.md:211]:
```suggestion
If the RHCOS image being used supports it,  the [KVM Qemu Guest Agent][qemu_guest_agent] may be used to enable optional
```

File: docs/user/openstack/install_upi.md [hunk 2]
```diff
@@ -207,6 +207,16 @@ $ openstack image create --container-format=bare --disk-format=qcow2 --file rhco
...
+access between OpenStack KVM hypervisors and the cluster nodes.
+
+To enable this feature, you must set hw_qemu_guest_agent=yes as a metadata parameter on the image.
```
Comment (mandre) [inline docs/user/openstack/install_upi.md:214]:
```suggestion
To enable this feature, you must add the `hw_qemu_guest_agent=yes` property to the image:
```

File: docs/user/openstack/install_upi.md (hunk 1 above)
Comment (adduarte) [reply docs/user/openstack/install_upi.md:211]: done

File: docs/user/openstack/install_upi.md (hunk 2 above)
Comment (adduarte) [reply docs/user/openstack/install_upi.md:214]: done

File: docs/user/openstack/install_upi.md [hunk 3]
```diff
@@ -207,6 +207,16 @@ $ openstack image create --container-format=bare --disk-format=qcow2 --file rhco
 
 **NOTE:** Depending on your OpenStack environment you can upload the RHCOS image as `raw` or `qcow2`. See [Disk and container formats for images](https://docs.openstack.org/image-guide/introduction.html#disk-and-container-formats-for-images) for more information.
 
+[qemu_guest_agent]: https://docs.openstack.org/nova/latest/admin/configuration/hypervisor-kvm.html 
+If the RHCOS image being used supports it,  the [KVM Qemu Guest Agent][qemu_guest_agent] may be used to enable optional
+access between OpenStack KVM hypervisors and the cluster nodes.
+
+To enable this feature, you must add the `hw_qemu_guest_agent=yes` poperty to the image:
```
Comment (mandre) [inline docs/user/openstack/install_upi.md:214]:
```suggestion
To enable this feature, you must add the `hw_qemu_guest_agent=yes` property to the image: 
```

**Review by mandre — COMMENTED** [review]
Just a little typo, looks good otherwise.

**Review by mandre — COMMENTED** [review]
/lgtm
/approve

**Comment by iamemilio** [comment]
Implements: OSASINFRA-2192

**Pull request merged**